apidoc/Titanium/UI/ScrollableView.yml

```diff
@@ -93,6 +93,22 @@ methods:
         summary: An integer index or <Titanium.UI.View> object to set as the current page.
         type: [Number, Titanium.UI.View]
 
+  - name: setIndicatorImageForPage
+    summary: Sets the indicator image for the specified page.
+    description: |
```

Comment:
Could probably benefit from an example in the docs

Reply:
Added.